package.json

```diff
@@ -42,6 +42,7 @@
     "@sentry/nextjs": "^6.19.3",
     "@sentry/react": "^6.19.2",
     "@sentry/tracing": "^6.19.2",
+    "@vegaprotocol/rest-clients": "^0.0.5",
```

Comment:
@dexturr  Rather than using the enums from rest-clients (or enums from graphql generation), would it not be better to import and use the enums that are actually derived from the wallet?